logviewer/build.gradle

```diff
@@ -0,0 +1,67 @@
+plugins {
+    id 'com.google.cloud.tools.jib' version '3.1.4'
```

Comment:
Please refactor to use the same style used in the main project.

Reply:
I don't see any difference to the buildscript in application or the parent script

Reply:
Right, sorry for that. Perfect.